src/test/java/com/prgrms/jpaweekly/order/OrderPersistenceTest.java

```diff
@@ -0,0 +1,112 @@
+package com.prgrms.jpaweekly.order;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.EntityManagerFactory;
+import jakarta.persistence.EntityTransaction;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Slf4j
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
```suggestion
@DataJpaTest
```

테스트하는 영역에 대해서 조금더 명확하게 구분해도 좋을 것 같습니다.